document-legacy/src/operation.rs

```diff
@@ -138,6 +139,7 @@ pub enum Operation {
 		layer_path: Vec<LayerId>,
 		delta: (f64, f64),
 		mirror_distance: bool,
+		reset_opposing_handle_lengths: Option<HashMap<u64, f64>>,
```

Comment:
But I don't think you should need to plumb any new arguments through in the operations. Can't you just call one (or perhaps multiple) existing operations to set the positions of the handle(s) to the intended locations, just like if the user had dragged them?

Reply:
I agree with Keavon here. I would suggest creating a new message type that triggers the handle position reset.

Reply:
I wasn't suggesting making a new type of operation. I was suggesting using the existing choice of operations that are already used to drag handles around with the user's cursor and use that to set the position to the coordinates determined within the logic of the Path tool code.

Reply:
Ah, I hadn't checked that. Can you make a new operation which specifies which points (based on index or something)? Because soon, we'll be replacing the entire document backend with a more data-centric approach so highly specialized messages like the reset one you added won't make sense in that data-centric implementation.

Reply:
Should I make a new operation or replace/modify `MoveSelectedManipulatorPoints`?

Reply:
Probably a new one.

Reply:
I'm sorry, there's a `MoveManipulatorPoint` message that can be used. I don't have to make a new message.

Reply:
Awesome! What's the status of this now? Is it functional and I should test it out and then give it another code review? If so, please mark it as not a draft ("ready for review" button).

Reply:
I have a few things to check, once done I'll mark it for review.